backend/curriculum_tracking/tests/test_activity_log_entry_creators_envoked.py

```diff
@@ -108,41 +111,74 @@ def test_add_review(self):
         self.assertEqual(LogEntry.objects.count(), 2)
 
 
-class log_card_review_requested_Tests(APITestCase, APITestCaseMixin):
+class log_card_review_requested_and_cancelled_Tests(APITestCase, APITestCaseMixin):
```

Comment:
This looks like you're replacing one test with another, instead of adding a new test. Is this intentional?

Reply:
Yup, the idea is to leverage the same setUp for both sets of tests

Reply:
@TheAbstract any other changes to request on this PR?